concourse/scripts/pipecleaner.py

```diff
@@ -27,9 +27,13 @@
 import sys
 import os
 import getopt
+import tempfile
+import subprocess
 
 
 class Pipecleaner(object):
+    def __init__(self, shellcheck_enabled):
+        self.shellcheck_enabled = shellcheck_enabled
```

Comment:
It does already.  https://github.com/alphagov/paas-cf/blob/master/.travis.yml#L22-L25

The flag was more for development as it's an incomplete feature